packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -184,10 +184,13 @@ const fetchDataset = async () => {
 				} else {
 					// We are assuming here there is only a single csv file. This may change in the future as the API allows for it.
 					image.value = undefined;
-					rawContent.value = await downloadRawFile(
-						props.assetId,
-						datasetTemp?.fileNames?.[0] ?? ''
-					);
+					// Tmep solution to avoid downloading raw file for netcdf files
```

Comment:
```suggestion
					// TODO = Temporary solution to avoid downloading raw NetCDF files, which can be massive
					// A better solution would be to check the size of an asset before downloading it, and/or 
					// downloading a small subset of it for presentation purposes.
```

Reply:
It doesn't download it - it straight up doesn't work since it expects csvs or something